In [2]:
import pandas as pd
 
 
url = "https://raw.githubusercontent.com/mricardo89/data-mining/main/Unit-II/datasets/titanic.csv"
df = pd.read_csv(url)

In [3]:
df.describe()

,PassengerId,Survived,Pclass,Age,SibSp,Parch,Fare
count,891.000000,891.000000,891.000000,714.000000,891.000000,891.000000,891.000000
mean,446.000000,0.383838,2.308642,29.699118,0.523008,0.381594,32.204208
std,257.353842,0.486592,0.836071,14.526497,1.102743,0.806057,49.693429
min,1.000000,0.000000,1.000000,0.420000,0.000000,0.000000,0.000000
25%,223.500000,0.000000,2.000000,20.125000,0.000000,0.000000,7.910400
50%,446.000000,0.000000,3.000000,28.000000,0.000000,0.000000,14.454200
75%,668.500000,1.000000,3.000000,38.000000,1.000000,0.000000,31.000000
max,891.000000,1.000000,3.000000,80.000000,8.000000,6.000000,512.329200


Pregunta A: 

75% = 31.000000 

Max = 512.329200

Diferencia de 41.3292 

Esto nos dice que hay una depresion en los precios, algunos pajaron tarifas más altas que otros.


Pregunta B:

Count PassengerId: 891.000000

Count Age: 714.000000

Diferencia de 177

Esto nos dice que 177 pasajeros no tiene un valor registrado en Age, lo que nos dice que la anamalia es que existen datos faltanes en la columna Age.


Pregunta C:

- Reemplazar los valores faltantes de Age pr la mediana de las edades
- Eliminar las filas de pasajeros que no tengan Age, aunque esto la cantidad de datos disponibles